content/en/database_monitoring/setup_oracle/rac.md

```diff
@@ -25,13 +25,35 @@ Database Monitoring provides deep visibility into your Oracle databases by expos
 
 Complete the following steps to enable Database Monitoring with your database:
 
-1. Configure the Agent for each RAC node by following the instructions for [self-hosted Oracle databases][7].
+Configure the Agent for each RAC node by following the instructions for [self-hosted Oracle databases][7].
 
-2. Add the following custom tag to each node:
-    ```yaml
-    tags:
-      - rac_cluster:your_cluster_name
-    ```
+You must configure the Agent for each RAC node, because the Agent collects information from every node separately by querying `V$` views. The Agent doesn't query and `GV$` views to avoid generating interconnect traffic. The collected data from each RAC node is aggregated in the frontend.
+
+```yaml
+init_config:
+instances:
+  - server: '<RAC_NODE_1>:<PORT>'
+    service_name: "<CDB_SERVICE_NAME>" # The Oracle CDB service name
+    username: 'c##datadog'
+    password: '<PASSWORD>'
+    dbm: true
+    tags:  # Optional
+      - 'service:<CUSTOM_SERVICE>'
+      - 'env:<CUSTOM_ENV>'
+  - server: '<RAC_NODE_2>:<PORT>'
+    service_name: "<CDB_SERVICE_NAME>" # The Oracle CDB service name
+    username: 'c##datadog'
+    password: '<PASSWORD>'
+    dbm: true
+    tags:  # Optional
+      - rac_cluster:<CLUSTER_NAME>
+      - 'service:<CUSTOM_SERVICE>'
+      - 'env:<CUSTOM_ENV>'
+```
+
+The Agent connects only to CDB. It queries the information about PDBs while connected to CDB. Don't create connections to individual PDBs.
```

Comment:
"Don't create connections to individual PDBs." - how do I ensure that I as a user am not doing this?

Reply:
Each database, CDB and PDB, has its own service name. A user has to configure the service name of the CDB with the parameter `service_name`.

Reply:
Ah I see, thanks